src/test/resources/testconfig.properties

```diff
@@ -4,5 +4,5 @@
 Verify.matlab.invokes.positive = MATLAB is invoking positive tests
 Verify.build.ignore.test.failure = Build Ignored test failure
```

Comment:
Why is Ignored capitalized? Do you mean "Build ignored the test failure"?

Reply:
Yes, Build ignored the test failure is correct. changing the capitalization